minimint-api/src/lib.rs

```diff
@@ -104,6 +106,11 @@ impl Amount {
         }
     }
 
+    pub fn from_str_in(s: &str, denom: Denomination) -> Result<Amount, ParseAmountError> {
+        let btc_amt = bitcoin::util::amount::Amount::from_str_in(s, denom)?;
+        Ok(Self::from(btc_amt))
```

Comment:
This will throw an error for sub-satoshi precise amounts [`ParseAmountError::TooPrecise`](https://github.com/rust-bitcoin/rust-bitcoin/blob/ed00c2aab90171fa74a812cd32fbac1fca9f678e/src/util/amount.rs#L215-L233).

Reply:
resolved by recent push